### PR TITLE
Spring cleaning

### PR DESCRIPTION
**Pull request by peterszerzo**
* [x] sidebar is removed - sidebar components have never really been used, they are too close in name to `Sidenav` components. I suggest removing them altogether - if we need them back, we can look at the implementation out of version control, but maybe start from scratch with real business needs in mind.
* [x] visual tests move to `emotion`.
* [x] visual tests can move to `Sidenav` components.
* [x] visualizations docs move to their respective folder.
* [x] remove website, instead, add splash screen to `styleguidist`.
* [x] fix minor breaks in components (`Breadcrumb`, `ButtonGroup`)

### CODE REVIEW

File: packages/visual-tests/src/App.tsx [hunk 1]
```diff
@@ -98,96 +98,89 @@ class App extends React.Component<{}, State> {
...
+                      }}
+                      label={test.title}
+                    />
+                  ))}
+                </SidenavHeader>
+              ))}
+            </Sidenav>
+          }
+          main={
+            <div style={{ height: "100%" }}>
```
Comment (TejasQ) [inline packages/visual-tests/src/App.tsx]: Is this necessary? Should Operational handle this out of the box?

File: packages/visual-tests/src/MarathonRenderer.tsx [hunk 2]
```diff
@@ -30,35 +30,31 @@ const Item = glamorous.li({
...
+  display: "inline-block",
+  marginLeft: 8,
+  "&::before": {
+    content: " → ",
+  },
+}))
 
 const MarathonRendererComponent = ({ results, ref }: MarathonRenderer) => (
-  <Div>
+  <div>
```
Comment (TejasQ) [inline packages/visual-tests/src/MarathonRenderer.tsx]: Können wir es als Fragment schreiben?

File: packages/visual-tests/src/App.tsx (hunk 1 above)
Comment (peterszerzo) [reply packages/visual-tests/src/App.tsx]: It used to, but now I'm having a hard time getting it to work. Care to lend me a hand?

File: packages/visual-tests/src/MarathonRenderer.tsx (hunk 2 above)
Comment (peterszerzo) [reply packages/visual-tests/src/MarathonRenderer.tsx]: Da hast du ja wohl Recht. Danke für den Vorschlag :)

File: packages/components/styleguide/Logo.tsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
+import * as React from "react"
```
Comment (TejasQ) [inline packages/components/styleguide/Logo.tsx]: Should this be called Logo?

File: packages/components/styleguide/OperationalLogo.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import * as React from "react"
```
Comment (TejasQ) [inline packages/components/styleguide/OperationalLogo.tsx]: Can we use `<Icon name="Operational" />` instead?

Comment (peterszerzo) [reply packages/components/styleguide/OperationalLogo.tsx]: This one is a slightly different format to accommodate the rotation.

File: packages/components/styleguide/Logo.tsx (hunk 3 above)
Comment (peterszerzo) [reply packages/components/styleguide/Logo.tsx]: You're right, we should call this LogoAndBanner for clarity.

Granted, the current way of adding the banner screen onto the website is a bit strange (going through a portal that is controlled through the logo component on the upper left corner), but everything else I've tried was hackier and less maintainable (copy-paste the implementation of styleguidist's page wrapper component and fix the typings manually).

File: packages/components/styleguide/OperationalLogo.tsx (hunk 4 above)
Comment (peterszerzo) [reply packages/components/styleguide/OperationalLogo.tsx]: Not if we want the rotation.

File: packages/visual-tests/src/App.tsx (hunk 1 above)
Comment (fabien0102) [reply packages/visual-tests/src/App.tsx]: Any update about this?

File: packages/visual-tests/src/MarathonRenderer.tsx (hunk 2 above)
Comment (fabien0102) [reply packages/visual-tests/src/MarathonRenderer.tsx]: No fragment?

File: packages/website/src/favicons/browserconfig.xml [hunk 5]
```diff
@@ -1,2 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<browserconfig><msapplication><tile><square70x70logo src="/ms-icon-70x70.png"/><square150x150logo src="/ms-icon-150x150.png"/><square310x310logo src="/ms-icon-310x310.png"/><TileColor>#ffffff</TileColor></tile></msapplication></browserconfig>
```
Comment (fabien0102) [inline packages/website/src/favicons/browserconfig.xml]: It can be nice to add a favicon to our styleguist ;)

**Pull request merged**